recipes/mmdet/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "mmdet" %}
+{% set version = "2.25.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/open-mmlab/mmdetection/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6642d593395a6c2d13bd1a574525477a8d08ef0fb7e67f9127a91b67193b06a9
+
+build:
+  #noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytorch
```

Comment:
is this really an install time dependency?

Reply:
Looking through the source ,it seems that all the stuff they included from pytorch in the build process just isn't used. maybe you can ask upstream to patch pytorch out of their build time requirements (your PR might look like a bunch of red lines).